util/statvar_dcid_generator.py

```diff
@@ -342,6 +355,32 @@ def _process_naics_range(range_str: str) -> str:
     return None
 
 
+def _soc_code_to_name(soc_val):
+    """Converts SOCv2018 codes to their industry using the SOC_MAP from
+    soc_codes_names.py
+
+    Args:
+        soc_val: A SOCv2018 string literal to process.
+          Expected syntax of soc_val - SOCv2018/{codes}
+    Returns:
+        A string with SOC code changed to it's occupation.
+        This string can be used in dcid generation. Returns None if the string
+        is empty.
+    """
+    if soc_val:
+        processed_str = 'SOC'
+
+        # Remove namespaces
+        soc_val = soc_val[soc_val.find(':') + 1:]
+
+        # Strip SOCv2018/
+        soc_val = soc_val.replace('SOCv2018/', '')
+
+        processed_str = processed_str + SOC_MAP[soc_val]
```

Comment:
What if soc_val is not present in SOC_MAP?

Reply:
Done. Have pushed the fix.